data/src/main/java/com/linkedin/data/schema/resolver/FileDataSchemaResolver.java

```diff
@@ -68,6 +68,12 @@
    */
   public static final String DEFAULT_EXTENSION = SchemaParser.FILE_EXTENSION;
 
+  /**
+   * The file path suffix for different types of schemas. Default is {@link InternalConstants#PEGASUS_DIR_IN_JAR}
+   * Ex "pegasus" for data or "extensions" for relationship extension schema files
+   */
+  private String _schemasPathSuffix = InternalConstants.PEGASUS_DIR_IN_JAR;
```

Comment:
It should be "prefix" not "suffix".

Reply:
its suffix to dir path and prefix to file name

Reply:
The path can be "/pegasus/foo/bar/..." calling it suffix doesn't sound right to me.

Reply:
removed suffix